variables.tf

```diff
@@ -14,48 +14,174 @@
+}
+
+variable "forseti_run_frequency" {
+  description = "Schedule of running the Forseti scans"
+  default     = "* */2 * * *"
+}
+
+#----------------#
+# Forseti server #
+#----------------#
+variable "server_type" {
+  description = "GCE Forseti Server role instance size"
+  default     = "n1-standard-2"
+}
+
+variable "server_region" {
+  description = "GCP region where Forseti will be deployed"
   default     = "us-central1"
 }
 
-variable "cloud_sql_region" {
-  description = "The Cloud SQL region"
+variable "server_boot_image" {
+  description = "GCE instance image that is being used, currently Debian only support is available"
+  default     = "ubuntu-os-cloud/ubuntu-1804-lts"
+}
+
+#----------------#
+# Forseti client #
+#----------------#
+variable "client_type" {
+  description = "GCE Forseti Client role instance size"
+  default     = "n1-standard-2"
+}
+
+variable "client_boot_image" {
+  description = "GCE Forseti Client role instance size"
+  default     = "ubuntu-os-cloud/ubuntu-1804-lts"
+}
+
+variable "client_region" {
+  description = "GCE Forseti Client role region size"
   default     = "us-central1"
 }
 
-variable "sendgrid_api_key" {
-  description = "The Sendgrid api key for notifier"
+#------------#
+# Forseti db #
+#------------#
+variable "cloudsql_region" {
+  description = "CloudSQL region"
+  default     = "us-central1"
+}
+
+variable "cloudsql_db_name" {
+  description = "CloudSQL database name"
+  default     = "forseti_security"
+}
+
+variable "cloudsql_db_port" {
+  description = "CloudSQL database port"
+  default     = "3306"
+}
+
+variable "cloudsql_proxy_arch" {
+  description = "CloudSQL Proxy architecture"
+  default     = "linux.amd64"
+}
+
+variable "cloudsql_type" {
+  description = "CloudSQL Instance size"
+  default     = "db-n1-standard-1"
+}
+
+#----------------#
+# Forseti bucket #
+#----------------#
+variable "storage_bucket_location" {
+  description = "GCS storage bucket location"
+  default     = "us-central1"
+}
+
+variable "bucket_cai_location" {
+  description = "GCS CAI storage bucket location"
   default     = ""
 }
 
-variable "notification_recipient_email" {
-  description = "Notification recipient email"
+variable "bucket_cai_lifecycle_age" {
+  description = "GCS CAI lifecycle age value"
+  default     = "14"
 }
 
-variable "gsuite_admin_email" {
-  description = "The email of a GSuite super admin, used for pulling user directory information."
+#---------#
+# Network #
+#---------#
+variable "vpc_host_network" {
+  description = "VPC host network"
+  default     = "default"
 }
 
-variable "project_id" {
-  description = "The ID of the project where Forseti will be installed"
+variable "vpc_host_subnetwork" {
+  description = "VPC subnetwork"
+  default     = "default"
 }
 
-variable "forseti_repo_url" {
-  description = "Foresti git repository URL"
-  default     = "https://github.com/GoogleCloudPlatform/forseti-security.git"
+variable "vpc_host_project_id" {
```

Comment:
What resources will be created in the host project, and what resources will be created in the forseti project?

Reply:
firewall resources need to have access to the host project